### PR TITLE
GCP - Specify instance on 'GCP - Firewall Remediation' playbook

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-dc.paloaltonetworks.com/browse/EXPANDR-7532

## Description
Added the *instance_name* input to allow the ability to specify *Google Cloud Compute* integration instance if you have multiple (optional).

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by johnnywilkes** [comment]
@ShirleyDenkberg , can you review, please?

**Comment by johnnywilkes** [comment]
@melamedbn  / @shmuel44 , I shared a demo video with you on gdrive

**Comment by johnnywilkes** [comment]
I dont think I can do anything for unit test errors, but please let me know

**Comment by shmuel44** [comment]
> I dont think I can do anything for unit test errors, but please let me know

Hi @johnnywilkes,
Thank you for your contribution!
For the unit test errors, try to merge from master...

**Comment by johnnywilkes** [comment]
@shmuel44 , i just bumped version and merged from master and still seeing errors

**Review by melamedbn — APPROVED** [review]

File: Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation.yml [hunk 1]
```diff
@@ -954,7 +966,14 @@ inputs:
   required: true
   description: The name of the GCP project associated with the instance and related objects.
   playbookInputQuery:
+- description: Google Cloud Compute integration instance to use if you have multiple (optional).
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation.yml:969]:
```suggestion
- description: Google Cloud Compute integration instance to use if you have multiple instances configured (optional).
```

File: Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation_README.md [hunk 2]
```diff
@@ -35,6 +35,7 @@ This playbook does not use any sub-playbooks.
 | RemotePort | The remote port that is publicly exposed to. | alert.remoteport | Required |
 | RemoteProtocol | The remote protocol that is publicly exposed to. |  | Required |
 | GcpProject | The name of the GCP project associated with the instance and related objects. | alert.asmcloud.Project | Required |
+| instance_name | Google Cloud Compute integration instance to use if you have multiple \(optional\). |  | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/Playbooks/GCP_-_Firewall_Remediation_README.md:38]:
```suggestion
| instance_name | Google Cloud Compute integration instance to use if you have multiple instances configured \(optional\). |  | Optional |
```

File: Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_13.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### GCP - Firewall Remediation
+
+Added the *instance_name* input to allow the ability to specify *Google Cloud Compute* integration instance if you have multiple (optional).
```
Comment (ShirleyDenkberg) [inline Packs/GCP-Enrichment-Remediation/ReleaseNotes/1_1_13.md:6]:
```suggestion
Added the *instance_name* input to allow the ability to specify a *Google Cloud Compute* integration instance if you have multiple instances configured (optional).
```

**Comment by ShirleyDenkberg** [comment]
@shmuel44 @melamedbn Doc review completed.

**Comment by shmuel44** [comment]
Hi @johnnywilkes ,
I merged the internal PR, and your update is now in the master branch of content repo.
Your update should be available in the marketplace within the next 72 hours.
Again, thank you very much for your contribution!

**Pull request merged**